relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -230,6 +231,10 @@ pub fn extract_transaction_metrics(
         sampling_metrics,
     )?;
 
+    if project_id.value() == 1 {
+        extract_span_metrics(aggregator_config, event, project_metrics)?;
```

Comment:
Why do we extract spans from within the `extract_transaction_metrics` function? 
Does it make sense actually to set the extraction up already in `relay-server/src/actors/processor.rs`, which mostly just calling same function just from the different place. 

The reason for this could be just to set up not a bit more which does not have to be moved in the later PRs.

Reply:
I'd like to do this in the processor. There's still a function in there that needs refactoring to not be tied up with the transaction metrics.